browser/vendor/borismus/webvr-polyfill.js

```diff
@@ -4587,11 +4589,17 @@ function MouseKeyboardVRDisplay() {
   this.isDragging_ = false;
 
   this.orientationOut_ = new Float32Array(4);
+
+  this.animationFilter_ = setInterval(function() {
+    this.filteredPhi_ = this.filteredPhi_ + (this.phi_ - this.filteredPhi_) / 8;
+    this.filteredTheta_ = this.filteredTheta_ + (this.theta_ - this.filteredTheta_) / 8;
+  }.bind(this), 1000/60);
 }
```

Comment:
setInterval is very unreliable for timing. Aren't parts of this script called on requestAnimationFrame? Maybe that could include some low-pass filtering for these two, also using the supplied timestamp for that call? Does angular acceleration affect inertia for this purpose, i.e. is inertia the same on drag start and drag end? I've found a natural-feeling control requires to take this into account too.